x-pack/plugin/wildcard/src/main/java/org/elasticsearch/xpack/wildcard/mapper/WildcardFieldMapper.java

```diff
@@ -551,6 +551,11 @@ protected void parseCreateField(ParseContext context) throws IOException {
         parseDoc.addAll(fields);
     }
 
+    @Override
+    protected String parseSourceValue(Object value) {
+        return value.toString();
```

Comment:
Same comment as the one for `keyword` field: shouldn't `ignore_above` be considered when parsing the source?